package.json

```diff
@@ -70,5 +70,11 @@
     "webpack-dev-server": "2.4.2",
     "wpcom": "5.3.0",
     "wpcom-proxy-request": "3.1.0"
+  },
+  "dependencies": {
+    "boom": "^7.2.0",
+    "cryptiles": "^4.1.1",
+    "fsevents": "^1.2.4",
+    "hawk": "^7.0.7"
```

Comment:
these probably belong above in `devDependencies` no?

also, why are we _adding_ these? I think we want to keep dependencies-of-dependencies out of `package.json` and update instead the primary dependencies which are pulling in the vulnerable version of `hoek`

Reply:
Wuuuuuuttt??!! Haha, I never even noticed I was adding these to `package.json` – I verified what I wanted to see in the package lock file and pushed 🙃Thanks, yes, I agree it should be just updating the dependencies themselves, will sort it out on Monday.